docs/user-guide/debugging-services.md

```diff
@@ -3,7 +3,7 @@
 
 An issue that comes up rather frequently for new installations of Kubernetes is
 that `Services` are not working properly.  You've run all your `Pod`s and
-`ReplicationController`s, but you get no response when you try to access them.
+`Deployment`s, but you get no response when you try to access them.
```

Comment:
This document stays k8s doesn't support interactive pods.  Would you update it while you are here?